AstarteDeviceSDKCSharp/Transport/MQTT/AstarteMqttTransport.cs

```diff
@@ -201,8 +201,18 @@ private Task OnDisconnectAsync(MqttClientDisconnectedEventArgs e)
 
         private void OnMessageReceive(MqttApplicationMessageReceivedEventArgs e)
         {
-            Trace.WriteLine("Incoming message: "
-            + Encoding.UTF8.GetString(e.ApplicationMessage.Payload));
+            object? payload = null;
+            if (e.ApplicationMessage.Payload == null)
+            {
+                Trace.WriteLine("Incoming message: Unsetted property "
+                + e.ApplicationMessage.Topic);
+            }
+            else
+            {
+                Trace.WriteLine("Incoming message: "
+                + Encoding.UTF8.GetString(e.ApplicationMessage.Payload));
+            }
```

Comment:
What do you think about this changes? It may be too early to tell that this is an unset property, so it is better to print a simple blank message if you want to debug this behavior.
```suggestion
                Trace.WriteLine("Incoming message on "
                + e.ApplicationMessage.Topic + " : " + Encoding.UTF8.GetString(e.ApplicationMessage.Payload ?? "{}"));
            
```

Reply:
try this but have the same problem, e.Application Message.Payload throws a null reference exception.

Reply:
Even if you add `?` before `Payload`?-> 
``` c#
Trace.WriteLine("Incoming message on "
                + e.ApplicationMessage.Topic + " : " + Encoding.UTF8.GetString(e.ApplicationMessage?.Payload ?? "{}"));
```

Reply:
I made changes.